packages/did/src/Did.utils.ts

```diff
@@ -264,10 +262,20 @@ export function isVerificationKey(
 export function isEncryptionKey(
   key: Partial<NewDidKey | DidKey> & Pick<NewDidKey | DidKey, 'type'>
 ): key is NewDidEncryptionKey | DidEncryptionKey {
-  const keyType = key.type.toLowerCase()
-  return encryptionKeyTypes.some((kt) => kt === keyType)
+  return encryptionKeyTypes.some((kt) => kt === key.type)
 }
 
+export type EncodedVerificationKey =
```

Comment:
Should these maybe be moved into the types package, for ease-of-use?
Otherwise it is hard to import them, when I need it externally

Reply:
we decided to rather reintroduce the `encodePublicKey` function.